content/en/news/releases/1.6.x/announcing-1.6.6/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Announcing Istio 1.6.6
+linktitle: 1.6.6
+subtitle: Patch Release
+description: Istio 1.6.6 patch release.
+publishdate: 2020-07-29
+release: 1.6.6
+aliases:
+    - /news/announcing-1.6.6
+---
+
+This release contains bug fixes to improve robustness. This release note describes
+what’s different between Istio 1.6.5 and Istio 1.6.6.
+
+{{< relnote >}}
+
+## Changes
+
+- **Optimized** `ResolveGatewayName` to greatly reduce the CPU usage and number of memory allocations.
```

Comment:
nit: `ResolveGatewayName` this is a super implementation detail, and ignores more important ones like #25613. I would consolidate this with L21 and L22 to something generic like `Optimized performance in scenarios with large number of gateways` and then just link all the issues/PRs.

Not a strong opinion though

Reply:
I think combining this and line 22 makes sense. Line 21 seems to have a separate affect which is that users reading through logs have very long log messages. The other two seem directly applicable to performance.